webserver/views/datasets.py

```diff
@@ -30,12 +29,43 @@ def view(id):
     )
 
 
+@datasets_bp.route("/<uuid:dataset_id>/evaluation")
+def view_latest_job(dataset_id):
+    ds = get_dataset(dataset_id)
+    jobs = db.dataset_eval.get_jobs_for_dataset(ds["id"])
+    if not jobs:
+        raise NotFound("Can't find any evaluation jobs for the specified dataset.")
```

Comment:
We shouldn't show a 404 here, the page doesn't look like the rest of the evaluation interface. Instead, do a 200, show the evaluation page interface, but have a message that says that this dataset hasn't been submitted for evaluation yet.